packages/assets-controllers/src/token-service.ts

```diff
@@ -58,6 +67,55 @@ function getTokenSearchURL(chainIds: CaipChainId[], query: string, limit = 10) {
   return `${TOKEN_END_POINT_API}/tokens/search?chainIds=${encodedChainIds}&query=${encodedQuery}&limit=${limit}`;
 }
 
+/**
+ * Get the trending tokens URL for the given networks and search query.
+ *
+ * @param chainIds - Array of CAIP format chain IDs (e.g., ['eip155:1', 'eip155:137', 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp']).
+ * @param sortBy - The sort by field.
+ * @param minLiquidity - The minimum liquidity.
+ * @param minVolume24hUsd - The minimum volume 24h in USD.
+ * @param maxVolume24hUsd - The maximum volume 24h in USD.
+ * @param minMarketCap - The minimum market cap.
+ * @param maxMarketCap - The maximum market cap.
+ * @returns The trending tokens URL.
+ */
+function getTrendingTokensURL(
+  chainIds: CaipChainId[],
+  sortBy?: SortTrendingBy,
+  minLiquidity?: number,
+  minVolume24hUsd?: number,
+  maxVolume24hUsd?: number,
+  minMarketCap?: number,
+  maxMarketCap?: number,
+) {
+  const encodedChainIds = chainIds
+    .map((id) => encodeURIComponent(id))
+    .join(',');
+  // Add the rest of query params if they are defined
+  const queryParams = new URLSearchParams();
+  if (sortBy) {
+    queryParams.append('sortBy', sortBy);
+  }
+  if (minLiquidity !== undefined) {
+    queryParams.append('minLiquidity', minLiquidity.toString());
+  }
+  if (minVolume24hUsd !== undefined) {
+    queryParams.append('minVolume24hUsd', minVolume24hUsd.toString());
+  }
+  if (maxVolume24hUsd !== undefined) {
+    queryParams.append('maxVolume24hUsd', maxVolume24hUsd.toString());
+  }
+  if (minMarketCap !== undefined) {
+    queryParams.append('minMarketCap', minMarketCap.toString());
+  }
+  if (maxMarketCap !== undefined) {
```

Comment:
Minor suggestion , to simplify this part, you could group these values under a single options parameter:
```
options: {
  sortBy?: SortTrendingBy;
  minLiquidity?: number;
  minVolume24hUsd?: number;
  maxVolume24hUsd?: number;
  minMarketCap?: number;
  maxMarketCap?: number;
} = {},
```
Then you can iterate through it like:

```
Object.entries(options).forEach(([key, value]) => {
  if (value !== undefined) queryParams.append(key, String(value));
});
```
This helps reduce repetition and makes it easier to extend later.